docs/system-admin-guide/integrations/gitlab-integration/README.md

```diff
@@ -80,7 +81,11 @@ https://myopenproject.com/webhooks/gitlab?key=4221687468163843
 > - Comments
 > - Issues events
 > - Merge request events
-> - Pipeline events
+> - Pipeline events 
+
+
+> **Note**: Please note that the *Pipeline events* part of the integration is still in the early stages. If you have any feedback on the *Pipeline events*, please let us know [here](https://github.com/btey/openproject-gitlab-integration/issues/43).
```

Comment:
We requested the author to archive the original plugin, since it was merged into OpenProject. (https://github.com/btey/openproject-gitlab-integration/pull/60)

I suggest using the new OPEN POINT work package to collect any feedback.

```suggestion
> **Note**: Please note that the *Pipeline events* part of the integration is still in the early stages. If you have any feedback on the *Pipeline events*, please let us know [here](https://community.openproject.org/wp/54574).
```